### PR TITLE
Replace only whole words when bumping version

### LINKED ISSUE #33: release.py version bumping is overeager
image

### PR DESCRIPTION
**Pull request by diaasami**
Closes #33

### CODE REVIEW

**Comment by ondrejbudai** [comment]
I like this method but I'm afraid that it can break if we e.g. use `Fedora 33` in a comment.

[RELEASING.md](https://github.com/osbuild/osbuild-composer/blob/main/RELEASING.md) for `osbuild-composer` suggests using `sed -i -E "s/(Version:\\s+)[0-9]+/\1$VERSION/" osbuild-composer.spec`. This regex basically does the change only on line that starts with `Version:`. We can do something similar but in Python. (Note that the `sed` script will break if used with point releases.)

File: release.py [hunk 1]
```diff
@@ -314,7 +314,8 @@ def bump_version(version, filename):
         content = file.read()
 
     # Maybe use re.sub in case the version appears a second time in the spec file
```
Comment (ochosi) [inline release.py:316]: This comment should be removed once the proper approach is implemented :)

**Comment by ochosi** [comment]
Why not iterate over the lines in the file and only do the change on that specific line? Then we could actually stick to the current method of stupidly replacing the string (because it also works for point releases).

**Review by ondrejbudai — APPROVED** [review]

**Review by ochosi — APPROVED** [review]
Nice - looks good! (Btw sorry if that was a misunderstanding, I'm not at all opinionated whether to do this iterating over lines or using re.sub - whatever works ;))

**Pull request merged**